src/condor_schedd.V6/qmgmt.cpp

```diff
@@ -3370,6 +3373,16 @@ UserCheck2(const ClassAd *ad, const char *test_user, const char *job_user)
 	if (is_same_user(test_user, job_user, opt, scheduler.uidDomain())) {
 		return true;
 	}
+	if ( ! ignore_domain_for_OwnerCheck) {
```

Comment:
Is this conditional reversed? We want to run this new code if we're ignoring the domain for OwnerCheck.

Reply:
yep.  it's reversed.